crates/bevy_input/src/keyboard.rs

```diff
@@ -86,16 +87,18 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 ///
 /// The event is consumed inside of the [`keyboard_input_system`]
 /// to update the [`Input<KeyCode>`](ButtonInput<KeyCode>) resource.
-#[derive(Event, Debug, Clone, Copy, PartialEq, Eq, Reflect)]
+#[derive(Event, Debug, Clone, PartialEq, Eq, Reflect)]
 #[reflect(Debug, PartialEq)]
 #[cfg_attr(
     feature = "serialize",
     derive(serde::Serialize, serde::Deserialize),
     reflect(Serialize, Deserialize)
 )]
 pub struct KeyboardInput {
-    /// The key code of the key.
+    /// The physical key code of the key.
     pub key_code: KeyCode,
+    /// The logical key of the input
+    pub logical_key: Key,
```

Comment:
maybe `key_logical` ? so nomenclature is similar for similar properties.

And rename `key_code`to `key_physical` ? Probably as a follow up, close to an existing one in https://github.com/bevyengine/bevy/issues/11052 ("Rename KeyCode to PhysicalKeyCode")

Reply:
or I guess we'll probably rename `key_code` to `physical_key`.

I just like the similar starting keystroks for similar properties, because it's autocompletion friendly, but I won't insist on it. (especially since winit uses `physical_key` and `logical_key`, so LGTM for now.